Formula/darktable.rb

```diff
@@ -0,0 +1,82 @@
+class Darktable < Formula
+  desc "Open source photography workflow application and raw developer"
+  homepage "https://www.darktable.org"
+  url "https://github.com/darktable-org/darktable/releases/download/release-2.2.4/darktable-2.2.4.tar.xz"
+  sha256 "bd5445d6b81fc3288fb07362870e24bb0b5378cacad2c6e6602e32de676bf9d8"
+
+  needs :openmp
+
+  depends_on "cmake"    => :build
+  depends_on "intltool" => :build
+  depends_on "jpeg"
+  depends_on "desktop-file-utils"
+  depends_on "exiv2"
+  depends_on "flickcurl"
+  depends_on "gcc"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gphoto2"
+  depends_on "graphicsmagick" => :optional
+  depends_on "gtk+3"
+  depends_on "gtk-mac-integration"
+  depends_on "json-glib"
+  depends_on "lensfun"
+  depends_on "librsvg"
+  depends_on "libsecret"
+  depends_on "libsoup"
+  depends_on "little-cms2"
+  depends_on "lua"
+  depends_on "ninja"
+  depends_on "openexr"
+  depends_on "openjpeg"
+  depends_on "osm-gps-map"
+  depends_on "pugixml"
+
+  fails_with "gcc" => 6
+  fails_with "gcc" => 7
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  def test_convert_to(extension)
+    expected_file = "image.#{extension}"
+    ohai "Converting raw to #{extension}"
+    system bin/"darktable-cli", "image.dng", "-o", expected_file
+    assert File.size?(expected_file)
+  end
+
+  def test_lua
+    require "open3"
+
+    ohai "Testing Lua support"
+    expected_string = "Hello Homebrew world!"
+    lua_code = <<-EOL
+    dt = require "darktable"
+    dt.print("#{expected_string}")
+    EOL
+
+    stdout = Open3.capture3(bin/"darktable-cli image.dng -o image.jpg --core --luacmd '#{lua_code}'")
+
+    assert_match expected_string, stdout[0]
+  end
+
+  test do
+    require "open-uri"
+
+    ohai "Downloading sample image"
+    File.open(testpath/"image.dng", "wb") do |fo|
```

Comment:
Instead make this a `resource` please; see e.g., [`flif.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/flif.rb).